pandas/tests/io/json/test_normalize.py

```diff
@@ -137,6 +137,11 @@ def max_level_test_input_data():
     ]
 
 
+@pytest.fixture
+def parse_metadata_fields_list_type():
```

Comment:
Can you just hardcode this in the function?

Reply:
@mroeschke like that?
`    def test_fields_list_type_normalize(self):
        parse_metadata_fields_list_type=[{"values": [1, 2, 3], "metadata": {"listdata": [1, 2]}}]
        result = json_normalize(
            parse_metadata_fields_list_type,
            record_path=["values"],
            meta=[["metadata", "listdata"]],
        )
        expected = DataFrame(
            {0: [1, 2, 3], "metadata.listdata": [[1, 2], [1, 2], [1, 2]]}
        )
        tm.assert_frame_equal(result, expected)`
        
        Should I push it? I did this as function just by looking other examples in the file.
        Let me know.

Reply:
Yes

Reply:
@mroeschke I just did it. But things looks different. I'm not sure if it was pushed properly. Can you check please? 
(My first PR for a big project)